Pachanga/Core/Firestore/MatchManager.swift

```diff
@@ -0,0 +1,145 @@
+//
+//  MatchManager.swift
+//  Pachanga
+//
+//  Created by Javier Alaves on 21/8/23.
+//
+
+import FirebaseFirestore
+import FirebaseFirestoreSwift
+import Foundation
+import SwiftUI
+
+struct Match: Codable {
+    
+    // props
+    let matchId: String
+    let dateCreated: Date
+    var location: String
+    var matchDate: Date
+    var players: [String] // combo of teamOne and teamTwo
+    var teamOne: [String] // tuple instead?
+    var teamTwo: [String]
+    var teamOneScore: Int
```

Comment:
Since there are always two teams as well, it could just be a single tuple where the left side corresponds to team 1 and the right side to team 2